build.gradle

```
@@ -43,19 +43,14 @@ task coverage(type: JacocoReport) {
 dependencies {
     String jUnitVersion = '5.4.0'
     String javaFxVersion = '11.0.2'
+    List<String> platforms = [ 'win', 'mac', 'linux' ]
+    List<String> jfxModules = [ 'javafx-base', 'javafx-graphics', 'javafx-controls', 'javafx-fxml' ]
 
-    implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'win'
-    implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'mac'
-    implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'linux'
-    implementation group: 'org.openjfx', name: 'javafx-controls', version: javaFxVersion, classifier: 'win'
-    implementation group: 'org.openjfx', name: 'javafx-controls', version: javaFxVersion, classifier: 'mac'
-    implementation group: 'org.openjfx', name: 'javafx-controls', version: javaFxVersion, classifier: 'linux'
-    implementation group: 'org.openjfx', name: 'javafx-fxml', version: javaFxVersion, classifier: 'win'
-    implementation group: 'org.openjfx', name: 'javafx-fxml', version: javaFxVersion, classifier: 'mac'
-    implementation group: 'org.openjfx', name: 'javafx-fxml', version: javaFxVersion, classifier: 'linux'
-    implementation group: 'org.openjfx', name: 'javafx-graphics', version: javaFxVersion, classifier: 'win'
-    implementation group: 'org.openjfx', name: 'javafx-graphics', version: javaFxVersion, classifier: 'mac'
-    implementation group: 'org.openjfx', name: 'javafx-graphics', version: javaFxVersion, classifier: 'linux'
+    jfxModules.each { mod ->
+        platforms.each { os ->
+            implementation group: 'org.openjfx', name: mod, version: javaFxVersion, classifier: os
```

It's also possible to write the statement as below which to me seems shorter and more compact.
```suggestion
            implementation "org.openjfx:$mod:$javaFxVersion:$os"
```

I used the current version to maintain the same flavor of dependency imports with the other modules.

---

Can also consider removing the repetition of the "javafx-" prefix in the jfxModules list by adding it within the implementation configuration:

```
List<String> jfxModules = ['base', 'graphics', 'controls', 'fxml']

    jfxModules.each { mod ->
        platforms.each { os ->
            implementation "org.openjfx:javafx-$mod:$javaFxVersion:$os"
        }
    }
```
Other than that LGTM!